docs/4.2/features/ssh_pam.md

```diff
@@ -0,0 +1,321 @@
+# Using Teleport with Pluggable Authentication Modules (PAM)
+
+Teleport SSH daemon can be configured to integrate with [PAM](https://en.wikipedia.org/wiki/Linux_PAM). This allows Teleport to create user sessions using PAM session profiles.
+
+Teleport only supports the `account` and `session` stack. The `auth` PAM module is
+currently not supported with Teleport.
+
+
+## Introduction to  Pluggable Authentication Modules
+
+Pluggable Authentication Modules (PAM) goes way to to 1995, when Sun Microsystems
```

Comment:
I'd probably say `Pluggable Authentication Modules (PAM) date back to 1995` instead